src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.html

```diff
@@ -1,116 +1,142 @@
 <div class="course-info-bar" *ngIf="exercise" id="exercise-header">
     <div class="row">
-        <div class="col general-info">
+        <div class="col-auto general-info">
             <h3>
                 <fa-icon *ngIf="displayBackButton" [icon]="faArrowLeft" (click)="onBackClick()" class="back-button me-2"></fa-icon>
+                <fa-icon *ngIf="exercise.type" [icon]="getIcon(exercise.type)" [ngbTooltip]="getIconTooltip(exercise.type) | artemisTranslate" container="body"></fa-icon>
                 <ng-content select="[pagetitle]"></ng-content>
                 <jhi-not-released-tag [exercise]="exercise"></jhi-not-released-tag>
             </h3>
-
+            <h4 class="row" *ngIf="exercise.maxPoints">
+                <span>
+                    {{ 'artemisApp.courseOverview.exerciseDetails.points' | artemisTranslate }}
+                    {{ exercise.maxPoints }}
```

Comment:
Optional:
Wouldn't it make sense to pass this as an argument to artemisTranslate?
This would mean that we could easily change this in the future by only having to edit the translation files and not the HTML code.

Would also apply to other occurences.

Reply:
The other occurrences are harder since they have whole spans for the badges and so one. Passing them as a parameter is not that clean in my opinion. 
And the I'd prefer to have this similar to the rest